tools/provision/darwin.sh

```diff
@@ -36,4 +36,5 @@ function main_darwin() {
   package google-benchmark
   package libmagic
   package sleuthkit
+  package aws-sdk-cpp
```

Comment:
Heads up that this won't work until https://github.com/Homebrew/homebrew-core/pull/109 is accepted. Until then, on Mac one can do `brew install https://raw.githubusercontent.com/zwass/homebrew-core/aws-sdk-cpp/Formula/aws-sdk-cpp.rb --with-static --without-http-client`.